.github/test-spec.yml

```
@@ -130,3 +130,6 @@
   - "nrf_security/**/*"
   - "crypto/**/*"
   - "nrf_rpc/**/*"
+
+"CI-wifi":
+  - "crypto/**/*"
```

Crypto just has the base libraries, nrf_security has most of the complexity of building the crypto libraries with mbedtls and PSA drivers.
```suggestion
  - "crypto/**/*"
  - "nrf_security/**/*"
```